assets/js/player.js

```diff
@@ -0,0 +1,13 @@
+function setClip(idClyp) {
+    const embedContainer = document.getElementById("player-embed-container");
+    embedContainer.innerHTML = '<iframe width="100%" height="265" src="https://clyp.it/' + idClyp + '/widget" frameborder="0"></iframe>';
+    
+    document.getElementById("player").classList.remove("player-hidden");
+
+    // Return false to block browser behaviour.
```

Comment:
Can use [Event.preventDefault()](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault) method instead to be more explicit.

Reply:
Unfortunately, since I don't think you can pass arguments using addEventListener without currying a function, returning false might be the most direct way to do this.

Reply:
You're already currying (kind of, it's technically just partial application) by using `onclick="return setClip(arg)"` in your HTML. This is equivalent to setting a callback function with only a single line (`return setClip(arg)`) as its body as explained [here](https://stackoverflow.com/a/21346394).

This is more of a style issue though so I can fix it myself after the PR is merged.